src/core/tests/Test_CoreMain.py

```diff
@@ -686,7 +685,6 @@ def test_auto_assessment_success_with_configure_patching_in_prev_operation_on_sa
         message = json.loads(substatus_file_data[1]["formattedMessage"]["message"])
         self.assertEqual(message["automaticOSPatchState"], Constants.AutomaticOSPatchStates.DISABLED)  # auto OS updates are disabled in RuntimeCompositor
         self.assertEqual(message["autoAssessmentStatus"]["autoAssessmentState"], Constants.AutoAssessmentStates.ENABLED)  # auto assessment is enabled
-
```

Comment:
nit: Please add the line breaks back. There are multiple scenarios tests in here, the line breaks where kept to ensure runtime.stop() is called at the end always. Line break highlights different sections in the code.

Applies to all other line breaks removed in here